core/src/main/java/org/elasticsearch/env/ESFileStore.java

```diff
@@ -73,13 +75,16 @@ private static String getMountPointLinux(FileStore store) {
         }
     }
     
-    /** Files.getFileStore(Path) useless here!  Don't complain, just try it yourself. */
-    static FileStore getMatchingFileStore(Path path, FileStore fileStores[]) throws IOException {
-        FileStore store = Files.getFileStore(path);
-        
+    /** 
+     * Files.getFileStore(Path) useless here!  Don't complain, just try it yourself. 
+     */
+    @SuppressForbidden(reason = "works around the bugs")
```

Comment:
just for kicks can we have `reason = "works around https://bugs.openjdk.java.net/browse/JDK-8034057"` it's just more obvious which bug is meant